src/content/pages/volunteers.mdx

```diff
@@ -95,13 +95,11 @@ There are always some tasks which we simply cannot plan beforehand. Based on wha
 
 # How to sign up
 
-Please acknowledge our [Data Privacy Policy](https://forms.gle/PCP7bXvdH1QhDBPn6) first, and then fill out the form below to register your interest in volunteering at EuroPython 2024!
-We will get back to you with more details and information on how to proceed.
-Thanks ❤️
+Thank you to all the enthusiastic Pythonistas who expressed interest in volunteering! With so many great applications, making a decision was tough this year.
+Unfortunately, we have already allocated all our volunteer tickets and are no longer able to offer you one at this time. Due to our limited capacity, we cannot accommodate everyone.
```

Comment:
I think we should swap the order of these two paragraphs. Make it obvious we're full first and then thank people.

Reply:
It was merged but we can still change it if you think it is needed. 
Maybe read it once on the website and let us know?